.github/workflows/spread-tests.yaml

```diff
@@ -0,0 +1,274 @@
+on:
+  workflow_call:
+    inputs:
+      runs-on:
+        description: 'A json list of tags to indicate which runner to use'
+        required: true
+        type: string
+      group:
+        description: 'The name of the group of backends, systems, tests, and rules'
+        required: true
+        type: string
+      backend:
+        description: 'The spread backend to use (for possible values, check spread.yaml)'
+        required: true
+        type: string
+      systems:
+        description: 'The spread system(s) to use (for possible values, check spread.yaml). If more than one, separate them with a space'
+        required: true
+        type: string
+      tests:
```

Comment:
While we use this for testing, technically those are "tasks". Perhaps we could use that naming for consistency.